src/main/java/fi/aalto/cs/apluscourses/BannerViewModel.java

```diff
@@ -0,0 +1,10 @@
+package fi.aalto.cs.apluscourses;
+
+import fi.aalto.cs.apluscourses.utils.observable.ObservableProperty;
+import fi.aalto.cs.apluscourses.utils.observable.ObservableReadWriteProperty;
+
+public abstract class BannerViewModel {
+  public final ObservableProperty<String> text = new ObservableReadWriteProperty<>("");
+
+  public abstract void update();
+}
```

Comment:
Do we need this kind of class? Can't the BannerView to be bound to any text (`ObservableProperty<String>`)?